samples/nrf9160/modem_shell/README.rst

```diff
@@ -379,6 +379,16 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+DK buttons
+==========
+
+The buttons have the following functions:
+
+Button 1:
+    * Raises a kill or abort signal. A long press of the button will kill or abort all supported running commands. You can abort commands ``iperf3`` (also with ``th``), ``curl``, ``ping`` and ``sock send``.
```

Comment:
No need for the bullets here. See, for example, https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/central_nfc_pairing/README.html

Reply:
Bullets removed from here.